lib/ansible/modules/cloud/ovirt/ovirt_auth.py

```diff
@@ -221,17 +238,23 @@ def main():
     elif state == 'absent':
         params = module.params['ovirt_auth']
 
+    url = params.get('url') or os.environ.get('OVIRT_URL')
+    username = params.get('username') or os.environ.get('OVIRT_USERNAME')
+    password = params.get('password') or os.environ.get('OVIRT_PASSWORD')
+    ca_file = params.get('ca_file') or os.environ.get('OVIRT_CAFILE')
```

Comment:
Since this might be nested-vars, or the user may supply a relative path. To make sure you handle this var well, also use `expanduser` and `expandvars`.